src/pages/patientView/clinicalInformation/ClinicalInformationSamplesTable.tsx

```diff
@@ -41,17 +44,16 @@ export default class ClinicalInformationSamplesTable extends React.Component<ICl
         return headerCells;
     }
 
-    public prepareData(sampleInvertedData: IConvertedSamplesData) {
+    public prepareData(sampleInvertedData: IConvertedSamplesData): ISampleRow[] {
 
-        const tableData: Array<any> = [];
+        const tableData: any[] = [];
```

Comment:
I think this should be 
tableData: ISampleRow[] = [];
It's possible that doing so would remove need to type function return (above) because ts can derive the type of the return.